### PR TITLE
Solve #237 bug

### PR DESCRIPTION
**Pull request by eamanu**
Solve indent problem in chaos.py

### CODE REVIEW

File: chaos.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def start_http_server():
     http_server_thread.start()
 
 def main():
-        logging.basicConfig(level=logging.DEBUG,
+    logging.basicConfig(level=logging.DEBUG,
                             format='%(asctime)s %(name)-12s %(levelname)-8s %(message)s',
```
Comment (ad-m) [inline chaos.py]: Do you can fix indent of that line too? See #246 for the right solution.

**Review by rhengles — CHANGES_REQUESTED** [review]
Has a merge conflict

**Comment by rhengles** [comment]
💩 !

**Comment by rhengles** [comment]
There was a commit 5 minutes ago, @chaosbot shouldn't have merged this, it should have reset the voting time!

**Comment by amoffat** [comment]
@rhengles you're right, it looks like there's a bug.  look at the top of the PR where it says `from unknown repo`.  This usually happens when you delete the repo backing a PR.  In order for github to keep the PR valid, the changeset lives somewhere... `unknown repo`.  Now, when @eamanu did the merge conflict fix, I would bet he did it in the browser, which means that that new conflict fix commit also lived at github's magic `unknown repo` as well, with the rest of his changes.  When chaos tries to find out how old a PR is for voting, it looks for the the repo backing a PR, and if it can't find it, [it defaults to](https://github.com/chaosbot/Chaos/blob/master/github_api/prs.py#L104-L112) the PR's creation date.

**Comment by amoffat** [comment]
i'll address this shortly in a new PR

**Pull request merged**